onward/app/views/fragments/mediaItem.scala.html

```diff
@@ -16,7 +16,10 @@
             }
             <h4 class="fc-item__title">
                 @content match {
-                    case a: Audio => { @a.metadata.webTitle }
+                    case a: Audio => {
+                        @fragments.inlineSvg("volume-high", "icon", List("yellow"))
```

Comment:
Is this ok to add, even for people not in the test?

Reply:
this template [isn't being used for audio](https://github.com/guardian/frontend/search?q=mediaitem&unscoped_q=mediaitem) at the moment so I think this is safe.
Also, all users will see this feature